test/e2e/e2e.spec.js

```diff
@@ -63,18 +63,18 @@ describe('/ (Verdaccio Page)', () => {
   });
 
   it('should match title with no packages published', async () => {
-    let text = await page.evaluate(() => document.querySelector('.container h1').textContent);
+    let text = await page.evaluate(() => document.querySelector('#help-card__title').textContent);
```

Comment:
I'm sorry for my lack of information...but we shouldn't use **const** here? the variable is not being assigned.